api/src/main/java/org/eclipse/microprofile/openapi/annotations/media/SchemaProperty.java

```diff
@@ -0,0 +1,371 @@
+/**
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ * Copyright 2017 SmartBear Software
```

Comment:
we probably don't need to include the SmartBear copyright for new file?

Reply:
Removed - I wasn't sure about this one since the file was largely a clone of Schema.java.